src/core/link-to-dfn.js

```diff
@@ -46,6 +44,20 @@ export function run(conf, doc, cb) {
         }
       }
     });
+    if (listOfDuplicateDfns.length > 0) {
+      var dfnsList = "";
+      for (var i = 0; i < listOfDuplicateDfns.length; i++) {
+        dfnsList += "["+ parseInt(i+1) + "](#" +
+          listOfDuplicateDfns[i].attr("id") + ")" + " ";
+      }
+      pub(
+        "error",
+        "Duplicate definitions of '" +
```

Comment:
Nit, use template string instead: 
```
`Duplicate definitions of '${title}' at: ${dfnsList}.`

Reply:
On it

Reply:
What I have done is that I am creating a `duplicate definitions` template string for each and every title